packages/vscode-extension/src/extension.ts

```diff
@@ -430,6 +441,33 @@ function registerChatParticipant(context: vscode.ExtensionContext) {
   context.subscriptions.push(generateManifestGUID);
 }
 
+/**
+ * Copilot Chat Participant for Office Add-in
+ */
+function registerOfficeChatParticipant(context: vscode.ExtensionContext) {
+  const participant = vscode.chat.createChatParticipant(
+    officeChatParticipantId,
+    officeChatRequestHandler
+  );
+  participant.iconPath = vscode.Uri.joinPath(context.extensionUri, "media", "teams.png");
+  participant.followupProvider = followupProvider;
+  participant.onDidReceiveFeedback((e) => handleFeedback(e));
+
+  context.subscriptions.push(
+    participant,
+    vscode.commands.registerCommand(CHAT_CREATE_OFFICE_SAMPLE_COMMAND_ID, chatCreateCommandHandler),
+    vscode.commands.registerCommand("fx-extension.openOfficeDevDocument", (...args) =>
```

Comment:
It could be `CommandKey.openOfficeDevDocument`?

Reply:
Since we are going to add the file tree structure when creating templates, we will use the CHAT_CREATE_SAMPLE_COMMAND_ID for the button and will remove this one.